frontend/src/proxy.js

```diff
@@ -0,0 +1,3 @@
+const proxy = "http://localhost:4000";
+
+export default proxy;
```

Comment:
This works! You can also create a `constants.js` to collect a bunch of stuff like this, if you find you need others